src/Apps/Auction/Routes/ConfirmBid/index.tsx

```diff
@@ -141,21 +148,30 @@ export const ConfirmBidRoute: React.FC<ConfirmBidProps> = props => {
     values: { selectedBid: number },
     actions: FormikActions<object>
   ) {
-    const bidderId = sale.registrationStatus.id
     const selectedBid = Number(values.selectedBid)
+    const possibleExistingBidderId: string | null = sale.registrationStatus
```

Comment:
I'm a bit confused here. The type for the mutation response states that the `registrationStatus` can be `null`, but seems like TS isn't picking up on that in this file? Check out the below screenshot:

<img width="1016" alt="Screen Shot 2019-11-07 at 3 26 30 PM" src="https://user-images.githubusercontent.com/1561546/68424988-26150500-0173-11ea-8ba2-a546b2ef2ba7.png">

Even when manually declaring that this var can be null-able...